tests/integration/ovs_test.py

```diff
@@ -41,6 +47,41 @@
         stp: true
 """
 
+OVS_SYSTEM_IFACE_NAME = 'eth1'
+OVS_INTERNAL_IFACE_NAME = 'ovs0'
+OVS_PORT_PREFIX = 'ovs-port'
+
+
+@pytest.fixture(scope='function')
+def ovs_bridge_up(eth1_up):
```

Comment:
It will be useful to add to the name the existence of a system and internal port attached.